src/main/java/org/mule/extension/email/internal/AbstractEmailConnection.java

```diff
@@ -151,18 +151,10 @@ private Properties buildSecureProperties(TlsContextFactory tlsContextFactory) th
       properties.setProperty(protocol.getSslCiphersuitesProperty(), join(cipherSuites, WHITESPACE_SEPARATOR));
     }
 
+    properties.setProperty(protocol.getSslEnableProperty(), "true");
```

Comment:
why?

Reply:
this is required by default if using tls